digits/job.py

```diff
@@ -56,6 +56,7 @@ def __init__(self, name, username, persistent = True):
         self._id = '%s-%s' % (time.strftime('%Y%m%d-%H%M%S'), os.urandom(2).encode('hex'))
         self._dir = os.path.join(config_value('jobs_dir'), self._id)
         self._name = name
+        self.group = group
```

Comment:
@gheinrich, I think moving the `group` to the `__setstate__` is the right thing to do. Having the group added at construction and load removes the need to checks in any other routes that might need the group. It is much cleaner. Thank you.

The `PICKLE_VERSION` bump doesn't facilitate any conditional code distinguishing jobs with and without `group` set, but as a matter of protocol, it seems like a fine thing to do.